crates/rpc-types/src/eth/log.rs

```diff
@@ -1,16 +1,14 @@
-use alloy_primitives::{Address, Bytes, B256, U256};
+use alloy_primitives::{B256, U256};
 use serde::{Deserialize, Serialize};
 
 /// Ethereum Log emitted by a transaction
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Default, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct Log {
-    /// Address
-    pub address: Address,
-    /// All topics of the log
-    pub topics: Vec<B256>,
-    /// Additional data fields of the log
-    pub data: Bytes,
+pub struct Log<T = alloy_primitives::LogData> {
+    #[serde(flatten)]
+    /// Consensus log object
+    pub inner: alloy_primitives::Log<T>,
```

Comment:
do we want getters for address, topics and data in `impl Log`?

Reply:
will add some for convenience